### PR TITLE
=pro make #21105 never ever happen again (OSGi breaking release)

### PR DESCRIPTION
**Pull request by ktoso**

### CODE REVIEW

**Comment by ktoso** [comment]
Resolves https://github.com/akka/akka/issues/21105

File: project/plugins.sbt [hunk 1]
```diff
@@ -13,7 +13,7 @@ addSbtPlugin("org.scalariform" % "sbt-scalariform" % "1.6.0")
 
 addSbtPlugin("com.typesafe.sbt" % "sbt-site" % "0.7.1")
 
-addSbtPlugin("com.typesafe.sbt" % "sbt-osgi" % "0.6.0")
+addSbtPlugin("com.typesafe.sbt" % "sbt-osgi" % "0.9.1")
```
Comment (ktoso) [inline project/plugins.sbt]: this version will blow up if we make some mistake <3

Comment (ktoso) [inline project/plugins.sbt]: It works if you're seeing `[info] Validating all packages are set private or exported for OSGi explicitly...`

**Comment by ktoso** [comment]
Failure was https://github.com/akka/akka/issues/21122

**Comment by ktoso** [comment]
Please review, we could release a 2.4.9 tomorrow I guess

**Comment by ktoso** [comment]
@drewhk @johanandren @2m Please review, we could release a 2.4.9 tomorrow I guess

**Comment by ktoso** [comment]
PLS BUILD

**Comment by johanandren** [comment]
LATM (looks awesome to me)

**Comment by ktoso** [comment]
I'll merge then. And we should likely release tomorrow, WDYT?

**Pull request merged**